qCC/db_tree/ccDBRoot.cpp

```diff
@@ -1700,6 +1704,7 @@ void ccDBRoot::selectByTypeAndName()
 	//For generic-only types the match type gets overridden and forced to
 	//false because exclusive match makes no sense!
 	bool exclusive;
+    bool regex;
```

Comment:
Also having other problems with the type selection. What kind of operator do we want to use? 

I mean if we disable the type search the regex/name search is not finding anything right now cause the filtering is always made before name-matching. I'll  try to put together a better proposal...